### PR TITLE
Remove followers run action flag

### PR DESCRIPTION
**Pull request by gbochora**
#### Summary
This PR removes the `broadcast to followers` flag from run data mode. There is no need for such a flag based on business requirements. 

#### Checklist

- ~~[ ] Telemetry updated~~
- ~~[ ] Gated by experimental feature flag~~
- [x] Unit tests updated

### CODE REVIEW

File: server/sqlstore/migrations.go [hunk 1]
```diff
@@ -1953,13 +1953,21 @@ var migrations = []Migration{
...
+		toVersion:   semver.MustParse("0.53.0"),
 		migrationFunc: func(e sqlx.Ext, sqlStore *SQLStore) error {
 			if e.DriverName() == model.DatabaseDriverMysql {
 				if err := addColumnToMySQLTable(e, "IR_Incident", "StatusUpdateBroadcastChannelsEnabled", "BOOLEAN DEFAULT FALSE"); err != nil {
 					return errors.Wrapf(err, "failed adding column StatusUpdateBroadcastChannelsEnabled to table IR_Incident")
 				}
-				if err := addColumnToMySQLTable(e, "IR_Incident", "StatusUpdateBroadcastFollowersEnabled", "BOOLEAN DEFAULT TRUE"); err != nil {
-					return errors.Wrapf(err, "failed adding column StatusUpdateBroadcastFollowersEnabled to table IR_Incident")
+				if err := dropColumnMySQL(e, "IR_Incident", "StatusUpdateBroadcastFollowersEnabled"); err != nil {
+					return errors.Wrapf(err, "failed droping column StatusUpdateBroadcastFollowersEnabled from table IR_Incident")
```
Comment (agarciamontoro) [inline server/sqlstore/migrations.go:1970]: Typo
```suggestion
					return errors.Wrapf(err, "failed dropping column StatusUpdateBroadcastFollowersEnabled from table IR_Incident")
```

File: server/sqlstore/migrations.go [hunk 2]
```diff
@@ -1968,8 +1976,8 @@ var migrations = []Migration{
 				if err := addColumnToPGTable(e, "IR_Incident", "StatusUpdateBroadcastChannelsEnabled", "BOOLEAN DEFAULT FALSE"); err != nil {
 					return errors.Wrapf(err, "failed adding column StatusUpdateBroadcastChannelsEnabled to table IR_Incident")
 				}
-				if err := addColumnToPGTable(e, "IR_Incident", "StatusUpdateBroadcastFollowersEnabled", "BOOLEAN DEFAULT TRUE"); err != nil {
-					return errors.Wrapf(err, "failed adding column StatusUpdateBroadcastFollowersEnabled to table IR_Incident")
+				if err := dropColumnPG(e, "IR_Incident", "StatusUpdateBroadcastFollowersEnabled"); err != nil {
+					return errors.Wrapf(err, "failed droping column StatusUpdateBroadcastFollowersEnabled from table IR_Incident")
```
Comment (agarciamontoro) [inline server/sqlstore/migrations.go:1980]: Typo
```suggestion
					return errors.Wrapf(err, "failed dropping column StatusUpdateBroadcastFollowersEnabled from table IR_Incident")
```

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
Looks good! I see that `dropColumnMySQL` and `dropColumnPG` only remove the column if it does exist, according to the code. But what happens when the exception `RAISE NOTICE 'Ignoring ALTER TABLE statement. Column "%s" does not exist in table "%s".';` is raised? Does that return an error? Or does it finish successfully?

I left a couple of typos I found below.

**Comment by gbochora** [comment]
> Looks good! I see that `dropColumnMySQL` and `dropColumnPG` only remove the column if it does exist, according to the code. But what happens when the exception `RAISE NOTICE 'Ignoring ALTER TABLE statement. Column "%s" does not exist in table "%s".';` from `dropColumnPG` is raised? Does that return an error? Or does it finish successfully?

Excellent question. It doesn't return an error. `NOTICE` messages are written to the database log, and the transaction is finished successfully.

**Review by agarciamontoro — APPROVED** [review]
> Excellent question. It doesn't return an error. NOTICE messages are written to the database log, and the transaction is finished successfully.

Perfect! Thanks for confirming :)

**Review by trilopin — APPROVED** [review]
Cool, Thanks @gbochora !

**Pull request merged**